packages/ember-testing/tests/helpers_test.js

```diff
@@ -656,21 +661,21 @@ test('`fillIn` takes context into consideration', function() {
   return wait();
 });
 
-test('`fillIn` focuses on the element', function() {
+QUnit.test('`fillIn` focuses on the element', function() {
   expect(2);
   var fillIn, find, visit, andThen, wait;
 
-  App.ApplicationRoute = Route.extend({
+  App.register('component:fill-in', Component.extend({
     actions: {
       wasFocused() {
         ok(true, 'focusIn event was triggered');
       }
     }
-  });
+  }));
 
-  App.IndexView = EmberView.extend({
-    template: compile('<div id="parent">{{input type="text" id="first" focus-in="wasFocused"}}</div>')
-  });
+  App.register('template:components/fill-in', compile('<div id="parent"><input type="text" id="first" onfocusin={{action "wasFocused"}}></div>'));
```

Comment:
Can you change this to use `{{input}}` and set this test to `test` from from `skip-it-glimmer`?